signingscript/sign.py

```diff
@@ -1020,3 +1128,48 @@ def verify_mar_signature(cert_type, fmt, mar, keyid=None):
         sig = widevine.generate_widevine_signature(signature, certificate, flags)
         fout.write(sig)
     return to
+
+
+async def sign_omnija_with_autograph(context, from_):
+    """Sign the omni.ja file specified using autograph and then
+    rebuild it using the signed meta-data and the original omni.ja
+    in order to facilitate the performance wins we do as part of the build
+
+    Args:
+        context (Context): the signing context
+        from_ (str): the source file to sign
+
+    Raises:
+        Requests.RequestException: on failure
+        SigningScriptError: when no suitable signing server is found for fmt
+
+    Returns:
+        str: the path to the signature file
+    """
+    raise NotImplementedError("Not Yet Written")
+    signed_out = tempfile.mkstemp(prefix="oj_signed", dir=context.config['work_dir'])
+    merged_out = tempfile.mkstemp(prefix="oj_merged", dir=context.config['work_dir'])
+    await sign_file_with_autograph(context, from_, "XXXCallek", to=signed_out)
+    await merge_omnija_files(orig=from_, signed=signed_out, to=merged_out)
```

Comment:
In iscript, I'm guessing we just need to add these two calls for any file named `omni.ja`, since we already have the dmg exploded? I hope so :)

What do you think about taking `vendored/` and `merge_omnija_files` and putting them into a standalone module that both signingscript and iscript can import?

Reply:
Hey there! I'm glad to see mozjar is now python3-compatible \o/ Mozapkpublisher will also use it because of https://bugzilla.mozilla.org/show_bug.cgi?id=1539933. 
Having a separate module would help this project too :)

Reply:
Agreed on seperate module, I'm holding off on doing it until scriptworker-monorepo though at this point